### ISSUE TITLE
Session Already Started During Installation 

### ISSUE THREAD

**Issue by ryangurn** [PodcastGenerator/PodcastGenerator#235]
**System Information**
Podcast Generator Version: 3.1
Webserver: Valet
PHP Version: 7.4.3

**Podcast Generator Config**

```
<?php
$podcastgen_version = "3.1"; // Version

$first_installation = 1584939537;

$installationKey = "b07m4iyf";

$scriptlang = "en_US";

$url = "http://podcastgenerator.test/";

$absoluteurl = "/Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/"; // The location on the server

$theme_path = "themes/default/";

$max_upload_form_size = "104857600"; //e.g.: "30000000" (about 30MB)

$upload_dir = "media/"; // "media/" the default folder (Trailing slash required). Set chmod 755

$img_dir = "images/"; // (Trailing slash required). Set chmod 755

$feed_dir = ""; // Where to create feed.xml (empty value = root directory). Set chmod 755

$max_recent = "all"; // How many file to show in the home page

$recent_episode_in_feed = "All"; // How many file to show in the XML feed (1,2,5 etc.. or "All")

$episodeperpage = 10;

$enablestreaming = "yes"; // Enable mp3 streaming? ("yes" or "no")

$freebox = "yes"; // enable freely customizable box

$enablepgnewsinadmin = "yes";

$strictfilenamepolicy = "no"; // strictly rename files (just characters A to Z and numbers) 

$categoriesenabled = "yes";

$cronAutoIndex = 1; //Auto Index New Episodes via Cron

$cronAutoRegenerateRSS = 1; //Auto regenerate RSS via Cron

$indexfile = "index.php";    // Path of the index file

$podcastPassword = "";       // Password to protect the podcast generator webpages, this will NOT protect the audio or XML files. Leave blank to disable.

#####################
# XML Feed stuff

$podcast_title = "Podcast Title";

$podcast_subtitle = "Subtitle";

$podcast_description = "A little description of your podcast.";

$author_name = "Podcast Generator User";

$author_email = "podcastgenerator@example.com";

$itunes_category[0] = "Arts"; // iTunes categories (mainCategory:subcategory)
$itunes_category[1] = "";
$itunes_category[2] = "";

$link = "?name="; // permalink URL of single episode (appears in the <link> and <guid> tags in the feed)

$feed_language = "en";

$copyright = "All rights reserved";   // Your copyright notice (e.g CC-BY)

$feed_encoding = "utf-8";

$explicit_podcast = "no"; //does your podcast contain explicit language? ("yes" or "no")

// END OF CONFIG

```  

**Expected behaviour**
A normally working GUI and podcast generator would be installed from scratch. 

**Actual behavior**
There are a variety of runtime errors that will display on the top of each page. 

```

Notice: Undefined index: podcastPassword in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/categories.php on line 17

Notice: Undefined index: podcastPassword in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/index.php on line 13

Notice: Undefined offset: 0 in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/index.php on line 48


Notice: Undefined index: feed_dir in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/core/feed_generator.php on line 15

Notice: Undefined index: itunes_category[1] in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/core/feed_generator.php on line 46

Notice: Undefined index: itunes_category[2] in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/core/feed_generator.php on line 49

Notice: Undefined index: feed_dir in /Users/ryangurnick/Desktop/GitHub/PodcastGenerator/PodcastGenerator/core/feed_generator.php on line 146
```

**How to reproduce**
Install from scratch seemingly?

**Comment by emilengler** [comment]
Thanks for the fix, like I said I will review this later today.
This is a development version and not a fixed release so such things might happen.
Also thanks for your contributions to the project recently <3

**Comment by ryangurn** [comment]
Not a problem, I will probably continue to help where I can

**Comment by emilengler** [comment]
Fixed in e8355b64400987b2f07e031e233e9c87c0dcacec
